src/cljs/district_voting/pages/vote_page.cljs

```diff
@@ -20,20 +20,48 @@
        :target :_blank}
    name])
 
-(def sort-options {:votes "DNT Votes"
-                   :gh-upvotes "Github upvotes"
-                   :gh-comments "Github comments"
-                   :latest "Latest"})
+(def sort-options {:dnt-votes {:title "DNT Votes"
+                               :reverse? true
+                               :cmp-fn :dnt-votes}
+                   ;;:upvotes "Github upvotes"
+                   :comments {:title "Github comments"
+                              :reverse? true
+                              :cmp-fn :comments}
+                   :created_at {:title "Latest"
+                                :reverse? true
+                                :cmp-fn :created_at}})
+
+(defn sort-pulldown [selected-value opts]
+  [ui/select-field
+   {:value @selected-value
+    :auto-width true
+    :on-change (fn [event idx value]
+                 (reset! selected-value (keyword value)))}
+   [ui/subheader {} "Sort by"]
+   (doall (map (fn [[k v]]
+                 ^{:key k}
+                 [ui/menu-item
+                  {:value k
+                   :key k
+                   :primary-text (:title v)}]) opts))])
 
 (defmethod page :route.vote/home []
   (let [votes (subscribe [:voting/candidates-voters-dnt-total :next-district])
         votes-total (subscribe [:voting/voters-dnt-total :next-district])
         loading? (subscribe [:voting-loading? :next-district])
         can-submit? (subscribe [:district0x/can-submit-into-blockchain?])
         vote-form (subscribe [:form.next-district/vote])
-        proposals (subscribe [::proposal-subs/list :next-district])
+        all-proposals (subscribe [::proposal-subs/list :next-district])
+        all-proposals-with-votes (reaction (doall (map (fn [p]
+                                                         (assoc p :dnt-votes (get @votes (:number p))))
+                                                       @all-proposals)))
         limit (r/atom 10)
-        sort-order (r/atom :votes)
+        sort-order (r/atom :dnt-votes)
+        proposals (reaction (let [sorted (sort-by (get-in sort-options
```

Comment:
Why don't you put this reaction as `reg-sub` into subs.cljs? Is it just temporarily here? Please keep all of them in subs.cljs, including `all-proposals-with-votes` above. Components should contain as least logic as possible